docs/src/doc/docs/user_guide/base-specific/frontend.md

```diff
@@ -69,3 +71,32 @@ In order to override a logo and style it accordingly a css file named `logo-styl
 
 
 For build system specific instructions please visit dedicated pages: [gradle](../gradle/usage.md#applying-plugins), [maven](../maven/usage.md#applying-plugins) and [cli](../cli/usage.md#configuration-options)
+
+## Custom HTML pages
+
+Dokka uses [FreeMarker](https://freemarker.apache.org/) template engine to render pages. 
+It takes templates from a folder that is set by a property `templatesDir`.
+To custom HTML output user can use a [default template](https://github.com/Kotlin/dokka/blob/master/plugins/base/src/main/resources/dokka/templates) as a basic.
+
+!!! note
+    To change page assets user can set properties `customAssets` and `customStyleSheets`.
+    Assets are handled by Dokka.
+
+Currently, there is one template file `base.ftl`. It defines general design of all pages to render.  
+
+Variables given below are available to a template:
+  - `${pageName}` - a page name
+  - `${footerMessage}` - a text is set by `footerMessage` property
+  - `${sourceSets}` - a nullable list of source set, only for multi-platform pages. Each source set has `name`, `platfrom` and `filter` properties.
+
+Also, Dokka-defined [directives](https://freemarker.apache.org/docs/ref_directive_userDefined.html) can be used:
+  - `<@content/>` - a main content
+  - `<@resources/>` - scripts, stylesheets 
+  - `<@version/>` - version (A version plugin replace this with a version navigator)
+  - `<@template_cmd name="...""> ...</@template_cmd>` - is used for stuff (`pathToRoot`, `projectName` are `name` parameter, local variables as well) that dependent on a root project. This is processed by a multi-module task that assembles a partial outputs from modules. 
+     Example:
+    ```
+    <@template_cmd name="projectName">
+       <span>${projectName}</span>
+    </@template_cmd>
+    ```
```

Comment:
Proposing to change `user` to `you` - it seems to be a common approach in user guides (looked at a few articles on kotlinlang and some coroutine pages)

Feel free to follow up on any proposed change, I can give more details as to why I'm suggesting it

Do tell me if I misunderstood what you meant in some of these

```suggestion
Templates are taken from the folder that is defined by the `templatesDir` property.
To customize HTML output, you can use the [default template](https://github.com/Kotlin/dokka/blob/master/plugins/base/src/main/resources/dokka/templates) as a starting point.

!!! note
    To change page assets, you can set properties `customAssets` and `customStyleSheets`.
    Assets are handled by Dokka itself, not FreeMaker.

Currently, there is only one template file `base.ftl`. It defines general design of all pages to render.  

Variables given below are available to every template:
  - `${pageName}` - page name
  - `${footerMessage}` - text that is set by the `footerMessage` property
  - `${sourceSets}` - a nullable list of source sets, only for multi-platform pages. Each source set has `name`, `platfrom` and `filter` properties.

Also, Dokka-defined [directives](https://freemarker.apache.org/docs/ref_directive_userDefined.html) can be used:
  - `<@content/>` - the main content
  - `<@resources/>` - scripts, stylesheets 
  - `<@version/>` - version ([versioning-plugin](https://kotlin.github.io/dokka/1.6.10/user_guide/versioning/versioning/) will replace this with a version navigator)
  - `<@template_cmd name="...""> ...</@template_cmd>` - is used for variables that dependent on the root project (such`pathToRoot`, `projectName`, `name`, local variables, etc) . This is processed by a multi-module task that assembles partial outputs from modules. 
     Example:
    ```
    <@template_cmd name="projectName">
       <span>${projectName}</span>
    </@template_cmd>
    ```
```

Reply:
> Proposing to change user to you - it seems to be a common approach in user guides (looked at a few articles on kotlinlang and some coroutine pages)
 
I do not like it neither but our documentation contains 'user' everywhere. I suppose It need to fix it everywhere in another PR.